src/centralized_telescope/setup.rs

```diff
@@ -0,0 +1,18 @@
+//! ALBA's Setup structure comprising, among others, the number of elements and internal parameters
+
+/// Setup output parameters
+#[derive(Debug, Clone, Copy)]
+pub struct Setup {
+    /// Approximate size of prover set to lower bound
+    pub set_size: u64,
+    /// Number of prover set's elements
```

Comment:
I'd change the comment to `The number of elements in a proof` to avoid confusion with `the total number of elements available to the prover at the proof generation`